### PR TITLE
fetching data is working

### PR DESCRIPTION
**Pull request by harshamsh**

### CODE REVIEW

File: routes/products.js [hunk 1]
```diff
@@ -27,19 +29,30 @@ router.post('/addproducts',async (req,res,next)=>{
 
 
 //// Fetch Product by productId
+
 router.get('/products/:id',(req,res,next)=>{
-    for (let index = 0; index < Products.length; index++) 
-    {
+    //feteching id parameter
+    fetchid = req.params.id;
```
Comment (jaga-live) [inline routes/products.js]:
```suggestion
    var productId = req.params.id;
```

File: routes/products.js [hunk 2]
```diff
@@ -27,19 +29,30 @@ router.post('/addproducts',async (req,res,next)=>{
...
+    //finding the value via productId field in the db
+    ProductModel.find(({productId:fetchid}),function(err,val){
```
Comment (jaga-live) [inline routes/products.js]:
```suggestion
   var product = await  ProductModel.findOne({productId})
   res.send(product)
```

Comment (jaga-live) [reply routes/products.js]: define a variable and store the data into that.. the variable can be used in other places

**Review by jaga-live — CHANGES_REQUESTED** [review]

**Comment by harshamsh** [comment]
defining-variable-db

**Comment by harshamsh** [comment]
defining-variable

**Pull request merged**